src/types/index.ts

```diff
@@ -0,0 +1,213 @@
+type TransformedEvent = {
```

Comment:
Replace all the references of `Object` in this file with the `Record<string, any>` type wherever the requirement is strictly an object.

Reply:
What about `errors` ?

Reply:
Or functions ?